web/profiles/custom/os2loop/modules/os2loop_upvote/src/Helper/FlagHelper.php

```diff
@@ -46,15 +62,24 @@ public function preprocessFlag(array &$variables) {
   /**
    * Add number of upvotes to flag.
    */
-  public function flagCounts(array &$variables) {
+  public function preprocessFlag(array &$variables) {
```

Comment:
Consider adding `Implements hook_preprocess_flag()` at the top of the comment to make it clear that this function actually does that.